config/services.yaml

```diff
@@ -8,6 +8,13 @@ parameters:
   env(ELASTIC_HOST): 'elasticsearch:9200'
   env(DATABASE_URL): 'mysql://root:api@mysql:3306/api?serverVersion=8.0'
 
```

Comment:
Compiler pass are great for bundles but are they for basic app? IMO, it adds unnecessary complexity. Maybe allow null arguments and falling back to empty array in the service would be enough

Reply:
IMO compiler pass is for bundles. I'm OK with default 500 in case you've not config.

Reply:
Compiler pass is knew from bundles, but can be used in application. It is really helpful for implementing eg. strategy pattern.

Reply:
Of course you can use them for implementing design patterns, but in this case, it's just a parameter. Another point is users have control over classes, if they decide to remove `exception_to_status` parameter, it's up to them to also remove its usage, don't you think ?

Reply:
Sure, i don't wanna fight with this. Doing CR in the night results sometimes weird ideas. I saw applications which used parameters configuration (a lot parameters) and when the grow up they are was really revengeful, so i just try avoid this. CR is great places for opinion exchanging, because every developer have a diffrent experience. Sometimes even weird ideas can be helpful for finding optimal solution.